packages/lib/i18n.ts

```diff
@@ -12,3 +12,14 @@ export const localeOptions = locales.map((locale) => ({
 export const defaultLocaleOption = localeOptions.find(
   (locale) => locale.value === i18n.defaultLocale
 ) as (typeof localeOptions)[number];
+
+// List of known RTL locales, it returnbs rtl or ltr for respective locales
+const rtlLocales = ["ar", "he"];
+
+export function getTextDirection(locale: string): "ltr" | "rtl" {
+  return rtlLocales.includes(locale) || rtlLocales.includes(locale.split("-")[0]) ? "rtl" : "ltr";
```

Comment:
Why are we doing extra things? In the future, if we add any new RTL locale, we will need to update it. Also, since we have access to the new intl object, why not use it? Sean also mentioned it.
Read more about it:- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getTextInfo#examples